common/app/views/fragments/analytics/google.scala.html

```diff
@@ -103,9 +103,14 @@
         @for(sponsorLogos <- listSponsorLogosOnPage(page)) {
           ga('@{trackerName}.set', 'dimension31', '@Html(sponsorLogos.mkString("|"))');
         }
+
         @for(brandingType <- Commercial.brandingType(page)) {
           ga('@{trackerName}.set', 'dimension42', '@brandingType.name');
         }
+
+        if(document.referrer.includes('oauth') && document.referrer.includes('facebook')) {
```

Comment:
Safe until someone changes the cookie expiry length in federation-api and forgets to update this 🙈